Library/Homebrew/cmd/audit.rb

```diff
@@ -1234,4 +1224,9 @@ def audit_urls
   def problem(text)
     @problems << text
   end
+
+  if ARGV.switch? "D"
+    Homebrew.inject_dump_stats!(FormulaAuditor, /^audit_*/)
+  end
 end
+
```

Comment:
Sorry; I've been addressing these feedback items individually without re-reading the whole diff on this. Will fix.